config.ts

```diff
@@ -2,6 +2,10 @@ import { goerli, optimismGoerli } from 'wagmi/chains';
 
 export const chains = [goerli, optimismGoerli];
 
+declare global {
```

Comment:
hmm, this should probably be relocated to a types file. in fact, can we remove and just make the coercion to number explicit? i'll give an example later in review

Reply:
yes, an example would be much appreciated. I tried with `as`, but it didn't work.

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/157/commits/283e41df1bf91e1a6eba8e6be69699ab7fa38a85